Userland/Libraries/LibWeb/CSS/StyleValue.h

```diff
@@ -150,6 +150,7 @@ namespace Filter {
 struct Blur {
     Optional<Length> radius {};
     float resolved_radius(Layout::Node const&) const;
+    bool operator==(Filter::Blur const& b) const;
```

Comment:
```suggestion
    bool operator==(Filter::Blur const&) const = default;
```
Now that there's an operator== on `AK::Variant` you should be able to default all these and remove the implementations from the .cpp.